input/schemas/paper_source_info.schema.yaml

```diff
@@ -78,11 +78,11 @@ properties:
           val: {type: number}
           err: {type: number, description: Statistical error}
 
-  spec:
+  spec: # in general: what to do with several values?
```

Comment:
For now, we only support one spectrum per paper. The total dataset and best-fit one should be picked.
We might expand this later, but for now, let's keep it simple and add more basic data on each source.